src/mbgl/text/glyph_atlas.cpp

```diff
@@ -147,27 +146,23 @@ void GlyphAtlas::addGlyphs(GlyphRequestor& requestor, const GlyphDependencies& g
         Entry& entry = entries[fontStack];
 
         for (const auto& glyphID : glyphIDs) {
-            loadedRanges.insert(getGlyphRange(glyphID));
+            optional<Glyph>& glyph = positions[glyphID];
```

Comment:
This line is really important to the logic in the tile worker, and it's effect is a little bit subtle. Maybe put a comment like:

// Make a glyph position entry even if we didn't get an SDF for the glyph
// This distinguishes between "not loaded yet" and "loaded but nothing to show"